lib/doro/entity.ex

```diff
@@ -37,7 +44,7 @@ defmodule Doro.Entity do
   def behaviors(nil), do: []
 
   def behaviors(entity = %Entity{}) do
-    entity.behaviors ++ behaviors(World.get_entity(entity.proto))
+    (entity.behaviors || []) ++ behaviors(World.get_entity(entity.proto))
```

Comment:
I'm kinda surprised about this, because of the [default](https://github.com/dorodango/doro/blob/bcb2a799b0b2b7a15e98d6698a40826e5e7a4fcc/lib/doro/entity.ex#L7) set in `entity.ex`.  But this is because behaviors submitted via a controller can be a non-array?